frontend/cypress/e2e/journeys.cy.js

```diff
@@ -0,0 +1,59 @@
+describe('Journeys page', () => {
+  beforeEach(() => {
+    cy.visit('/journeys');
+  });
+
+  it('should have a title', () => {
+    cy.get('.m-journey__title h1')
+      .first()
+      .should('contain', 'Discover Journeys');
+  });
+
+  it('should there be a list of 5 journeys', () => {
+    cy.get('.m-journey__gridelement').should('have.length', 5);
+  });
```

Comment:
This test won't be relevant anymore when the admins start to create new journeys. The ones below as well. The admins may change the names or order of the journeys. For now, we can probably keep it that way.

What would you propose in the future?

Reply:
I'm considering two options: first, having a mock and intercepting the endpoint of the journey; second, intercept the request and make it dynamic and dependable to the endpoint result

Reply:
Are you gonna make these changes in this PR?